.teamcity/src/main/kotlin/model/CIBuildModel.kt

```diff
@@ -42,7 +42,7 @@ enum class StageName(val stageName: String, val description: String, val uuid: S
 
 private val performanceRegressionTestCoverages = listOf(
     PerformanceTestCoverage(1, PerformanceTestType.per_commit, Os.LINUX, numberOfBuckets = 40, oldUuid = "PerformanceTestTestLinux"),
-    PerformanceTestCoverage(6, PerformanceTestType.per_commit, Os.WINDOWS, numberOfBuckets = 5, failsStage = false),
+    PerformanceTestCoverage(6, PerformanceTestType.per_commit, Os.WINDOWS, numberOfBuckets = 10, failsStage = false),
```

Comment:
🤔 Even with ten buckets, I am somewhat surprised it didn't split the NowInAndroid bucket. Could you check the runtimes why that is the case? Is something else going on there?

Another thing I am worried about is that we might be scarce on Windows agents. Adding this split depends a lot on how expensive it is to generate the test project. Not sure where we are there with the different projects on Windows.

All in all, I'd still go ahead with this and try it out.